requirements/base.rqr

```diff
@@ -4,4 +4,5 @@ mecab-python3
 absl-py
 dpath
 ipadic
-scipy
\ No newline at end of file
+scipy
+pytrec_eval
```

Comment:
```suggestion
```
This should be moved to `requirements/tests.rqr`
See the other comment for full explanation

Reply:
done